### PR TITLE
adds ethereal mutation toxin and a recipe for it

### PR DESCRIPTION
**Pull request by boodaliboo**
### Intent of your Pull Request
currently theres no way to become a ethereal even with strange seeds so i added a chemical recipe using liquid electiricty and a green extract to become one

#### Changelog

:cl:  
rscadd: Added ethereal mutation toxin and recipe    
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/recipes/slime_extracts.dm [hunk 1]
```diff
@@ -70,6 +70,14 @@
 	required_other = TRUE
 	required_container = /obj/item/slime_extract/green
 
+/datum/chemical_reaction/slime/slimeethereal
+	name = "Ethereal Mutation Toxin"
+	id = "etherealarerealmuttoxin"
+	results = list(/datum/reagent/mutationtoxin/ethereal = 1)
+	required_reagents = list(/datum/reagent/consumable/liquidelectricity = 1)
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm:77]: Maybe teslium instead?

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/recipes/slime_extracts.dm (hunk 1 above)
Comment (boodaliboo) [reply code/modules/reagents/chemistry/recipes/slime_extracts.dm:77]: yeah that sounds better ill change it in a bit

Comment (boodaliboo) [reply code/modules/reagents/chemistry/recipes/slime_extracts.dm:77]: was just thinking that liquid electricty was available already from high power energy bar and its also the blood of ethereals

**Comment by boodaliboo** [comment]
so do people want it to be teslium or liquid elec

**Comment by alexkar598** [comment]
Not interested and it seems noone else is either

**Pull request closed without merging**